Formula/dps8m.rb

```diff
@@ -0,0 +1,31 @@
+class Dps8m < Formula
+  desc "Simulator for the Multics dps-8/m mainframe"
+  homepage "http://ringzero.wikidot.com"
+  url "https://downloads.sourceforge.net/project/dps8m/Release%201.0/source.tgz"
+  version "1.0"
+  sha256 "51088dd91de888b918644c431eec22318640d28eb3050d9c01cd072aa7cca3c7"
+
+  depends_on "libuv"
+
+  def install
+    bin.mkpath
```

Comment:
@ilovezfs The bug was reported upstream https://sourceforge.net/p/dps8m/mailman/message/35960505/